### PR TITLE
CORE-12019 Capture CPI build errors simple impl from Graham

### PR DESCRIPTION
**Pull request by egabrovski-r3**
Moved from https://github.com/corda/CSDE-cordapp-template-kotlin/pull/57

### CODE REVIEW

File: buildSrc/src/main/java/com/r3/csde/BuildCPIsHelper.java [hunk 1]
```diff
@@ -215,6 +215,12 @@ private void createCorDappCPI() throws IOException, InterruptedException, CsdeEx
 //        }
 //        fileWriter.close();
 
+// todo: in the meantime, here is a simple working impl
```
Comment (yarivus) [inline buildSrc/src/main/java/com/r3/csde/BuildCPIsHelper.java]: Looking at the previous code and the new one which is simple and does the job - why is it remaining as TODO ?
Why are we leaving the previous commented out code ?

**Review by yarivus — CHANGES_REQUESTED** [review]

File: buildSrc/src/main/java/com/r3/csde/BuildCPIsHelper.java (hunk 1 above)
Comment (egabrovski-r3) [reply buildSrc/src/main/java/com/r3/csde/BuildCPIsHelper.java]: This is a copy-paste from https://github.com/corda/CSDE-cordapp-template-kotlin/pull/29, a change that I have not done, or look into.
There might be a better solution, or this impl might have missed something ... so I decided to keep the history and the context in the code
please confirm if you prefer to delete the old code and the todo?

Comment (yarivus) [reply buildSrc/src/main/java/com/r3/csde/BuildCPIsHelper.java]: @mattbradburyr3 please confirm we can delete the old commented out code along with the TODO comment

Comment (mattbradburyr3) [reply buildSrc/src/main/java/com/r3/csde/BuildCPIsHelper.java]: All of this code is being refactored into the gradle plug in. There is a todo in the gradle code to work out how to handle the error stream which references this PR. Hence we can delete the commented out code.

Comment (egabrovski-r3) [reply buildSrc/src/main/java/com/r3/csde/BuildCPIsHelper.java]: done

**Comment by egabrovski-r3** [comment]
qa-regpack-csde tests are green

**Review by yarivus — APPROVED** [review]
LGTM (Thanks 👍 )

**Pull request merged**